cartridge-cli.lua

```diff
@@ -2434,6 +2474,9 @@ local function app_pack(args)
     else
         die("Unknown package type: %s", args.type)
     end
+
+    -- clean build directory
+    fio.rmtree(pack_state.build_dir)
```

Comment:
In case if application die, we don't remove build_dir

I think `pack_*` functions should respect `nil, err` convention. And we will have one entry point where can process an error

Reply:
You can create a table with "handlers" e.g.
```lua
local handlers = {
   ['docker'] = pack_docker,
   ['rpm'] = pack_rpm,
   ...
}

local pack = handlers[mode]
if pack == nil then
    die()
end

local ok, err = pack(args)
fio.rmtree(builddir)
if not ok then
    die(err)
else
   info('Success!')
end
```

Reply:
I like your idea, but I sure that it should be done in separated patch.

Moreover, I suggest adding named constants for distribution types to use them in code and error messages.

Reply:
See #127